src/iam.ts

```diff
@@ -88,7 +88,8 @@ export interface IClaimRequest extends IMessage {
 }
 
 export interface IClaimIssuance extends IMessage {
-  issuedToken: string;
+  issuedToken?: string;
+  onChainProof?: string;
```

Comment:
```suggestion
  // issuedToken is is only provided in the case of off-chain role
  issuedToken?: string;
  // onChainProof is only provided in case of on-chain role
  onChainProof?: string;
```